src/llvm/ext/find-llvm-config.ps1

```diff
@@ -0,0 +1,24 @@
+$LLVM_CONFIG = $env:LLVM_CONFIG -or (Get-Command "llvm-config" -ErrorAction SilentlyContinue)
+
+if (-not $LLVM_CONFIG) {
+    $llvm_config_version = $(llvm-config --version 2>$null)
```

Comment:
Why doesn't this line use `Get-Command`?

Reply:
Not proficient in powershell - actually used chat-gpt 4 to generate it -  I just know a solution is needed for this bash script in windows